cmd/server/cadence/cadence.go

```diff
@@ -135,8 +138,18 @@ func BuildCLI() *cli.App {
 		{
 			Name:    "start",
 			Aliases: []string{""},
-			Usage:   "start cadence notification service",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "service",
+					Value: "",
```

Comment:
Sorry I should have put it more details in the ticket -- 
can we put default values as "receiver,notifier" here? 
By default this command will start both. And if user like, then can choose to start only one of them.

this is the same as the server repo: https://github.com/uber/cadence/blob/b7d2c776469418da99b117e2fea585c61cb611e5/cmd/server/cadence/cadence.go#L214